Add supplystack/codacy-coverage-reporter Plugin

BitProcessor opened the pull request.

appleboy commented on content/SupplyStack/codacy-coverage-reporter/index.md:
```diff
@@ -0,0 +1,44 @@
+tags: [ coverage ]
+logo: codacy.svg
+repo: supplystack/codacy-coverage-reporter
+image: supplystack/codacy-coverage-reporter:6.0.6
```
remove `6.0.6` version?

BitProcessor replied:
done

BitProcessor commented on content/SupplyStack/codacy-coverage-reporter/index.md:
```diff
@@ -0,0 +1,44 @@
+
+steps:
+- name: codacy
+  image: supplystack/codacy-coverage-reporter:6.0.6
```
Also removed version here

appleboy commented:
Need another maintainer approval @tboerger

tboerger commented:
Oh, deleted the repository?

BitProcessor commented:
Hi Thomas,

We ditched Codacy in favour of Sonar. So no further maintenance to the plugin will be performed.
That’s why I deleted the repo and PR.

Best regards,
Maarten





> On 17 Dec 2019, at 14:40, Thomas Boerger <notifications@github.com> wrote:
> 
> Oh, deleted the repository?
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub <https://github.com/drone/drone-plugin-index/pull/244?email_source=notifications&email_token=AMCK5TJHS6VOBBRIUFSZ5WDQZDJDLA5CNFSM4JOROAP2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEHCMYNA#issuecomment-566545460>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AMCK5TLFNK3T6GU5R6ACL5TQZDJDLANCNFSM4JOROAPQ>.
>
